beaumont/config_HOF_2021.json

```diff
@@ -0,0 +1,188 @@
+{
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
+      }
+    },
+    {
+      "dataType": "dwi",
+      "modalityLabel": "dwi",
+      "criteria": {
+        "SeriesDescription": "dwi_acq-pa_standard"
+      }
+    },
+    {
+      "dataType": "fmap",
+      "modalityLabel": "epi",
+      "criteria": {
+        "SeriesDescription": "dwi_acq-ap_standard"
+      },
+      "IntendedFor": 1
+    },
+    {
+      "dataType": "anat",
+      "modalityLabel": "GRE",
+      "customLabels": "acq-PDw_flip-1_mt-off",
+      "criteria": {
+        "SeriesDescription": "gre_3d_PDw"
+      },
+      "sidecarChanges": {
+        "MTState": false
+      }
+    },
+    {
+      "dataType": "anat",
+      "modalityLabel": "GRE",
+      "customLabels": "acq-MTw_flip-1_mt-on",
+      "criteria": {
+        "SeriesDescription": "gre_3d_MTw"
+      },
+      "sidecarChanges": {
+        "MTState": true
+      }
+    },
+    {
+      "dataType": "anat",
+      "modalityLabel": "GRE",
+      "customLabels": "acq-T1w_flip-2_mt-off",
+      "criteria": {
+        "SeriesDescription": "gre_3d_T1w"
+      },
+      "sidecarChanges": {
+        "MTState": true
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-1_flip-1_part-mag_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0075,
```

Comment:
I would use EchoNumber instead of EchoTime since it can slightly vary.

Reply:
I change with echo numbers, but note that echo 1 and 2 are different between the 2 sequences. That's why I went with `echo-3` and `echo-4` to make sure we _remember_ they differ.